src/ReleaseNotesUtil/Program.cs

```diff
@@ -88,32 +86,55 @@ private static void GetRulesJson(string nugetInstalledPackagesPath, string versi
             IEnumerable<RuleInfo> removedRules =
                 oldContent.Rules
                     .Where(r => !newRulesById.ContainsKey(r.Id));
+            IEnumerable<RuleInfo> changedRules =
+                newContent.Rules
+                    .Where(r => oldRulesById.TryGetValue(r.Id, out RuleInfo oldRule)
+                                && r.IsEnabledByDefault != oldRule.IsEnabledByDefault);
```

Comment:
Should we compare any other rule metadata? DiagnosticDescriptor itself seems to equatable: http://source.roslyn.io/#Microsoft.CodeAnalysis/Diagnostic/DiagnosticDescriptor.cs,14

Reply:
I figured enabled-by-default would be the only important thing people would care about, unless you think people want to see fixers or something?

I didn't think it'd be all that important to note changes to descriptions or tags or what not.

Reply:
Sounds reasonable!